src/Starlight.Server/Hooks/HookCaller.cs

```diff
@@ -65,6 +68,83 @@ public async Task<HandleResult> OnQuickStachAsync(OnQuickStackArgs args)
             return HandleResult.Continue();
         }
 
+
+        private bool currentGameMenuState;
+        public async Task InvokeGameUpdateAsync()
```

Comment:
```suggestion
        public async Task OnGameUpdateAsync()
```

Inconsistent naming. Why are we adding `Invoke` here?